driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -744,6 +745,22 @@ public Builder withPort(int port) {
         }
 
         /**
+         * Allows client to connect to server using latest development protocol version,
+         * which is currently in beta.
```

Comment:
Maybe add here some explanation about what are the implications of calling this method (i.e. the flag `USE_BETA` would be added to all outgoing messages) and also warn that it only makes sense for protocol versions >= 5.